styles.css

```diff
@@ -463,27 +596,53 @@ textarea {
 #four {
   content: "";
   position: relative;
-  width: 90.81px;
-  height: 130px;
-  left: 295px;
-  top: -220px;
-  background-image: url('img/illustration.png');
-  background-size: contain; 
-  background-position: center center; 
-  background-repeat: no-repeat; 
+  width: 313.9px;
+  height: 448.7px;
+  left: 900px;
+  top: -280px;
+  background-image: url('img/contact\ form-illustratoin-3-desktop.png');
+  background-size: contain;
+  background-position: center center;
+  background-repeat: no-repeat;
+  z-index: 3;
+}
+
+#five {
+  content: "";
+  position: relative;
+  width: 235.78px;
+  height: 261.79px;
+  left: -800px;
+  top: -200px;
+  background-image: url('img/illustration-1-contact-form-desktop.png');
+  background-size: contain;
+  background-position: center center;
+  background-repeat: no-repeat;
+  z-index: 3;
+}
+
+#six {
+  content: "";
+  position: relative;
+  width: 130.28px;
+  height: 105.85px;
+  left: -1000px;
+  top: -180px;
+  background-image: url('img/illustration-2-contact-form-desktop.png');
+  background-size: contain;
+  background-position: center center;
+  background-repeat: no-repeat;
   z-index: 3;
 }
 
 @media only screen and (min-width: 375px) {
   body {
     font-size: 16px;
   }
-
-
 }
 
 @media only screen and (min-width: 768px) {
   body {
     font-size: 25px;
```

Comment:
@joskalenda I have separate pull requests both for the mobile version and the desktop version, this one was for the desktop version

Reply:
- [ ] It is not supposed to be so, your desktop version should have the mobile version working perfectly because you have started from it, the smaller modifications you should bring on the mobile version to have the desktop version, separating the pull request is not a problem but before you start working on the desktop version, you have to make sure your version one of mobile code is included

Reply:
@joskalenda how do I include the mobile code and make sure it loads up once the screen size is reduced?